File: pocs/android_only_m2_j17jars/app/src/main/java/bisq/android/main/user_profile/UserProfileView.java
package bisq.android.main.user_profile;

public class UserProfileView {
    private final UserProfileController userProfileController;
    private final UserProfileModel model;

    public UserProfileView(UserProfileController userProfileController, UserProfileModel model) {
        this.userProfileController = userProfileController;
        this.model = model;
    }

    public void initialize() {
       /* model.getNickName().addObserver(value -> nickNameTextField.setText(value));
        model.getNym().addObserver(value -> nymTextField.setText(value));*/
    }
}

File: pocs/android_only_m2_j17jars/app/src/main/java/bisq/android/main/user_profile/UserProfileView.kt
package bisq.android.main.user_profile

class UserProfileView(
    private val userProfileController: UserProfileController,
    private val model: UserProfileModel
) {
    fun initialize() {
        /* model.getNickName().addObserver(value -> nickNameTextField.setText(value));
        model.getNym().addObserver(value -> nymTextField.setText(value));*/
    }
}
